viewer/static/js/app/reducers/metric-metadata-reducer.js

```diff
@@ -15,7 +15,7 @@ const getHoverStrings = (metadata) => {
   for (const id in metadata) {
     if (metadata.hasOwnProperty(id)) {
       var tooltip = metadata[id].tooltip;
-      strings['id-' + id] = tooltip ? tooltip : '{pop}: x={x} {xunit}, y={y}%, n={n}';
+      strings['id-' + id] = tooltip ? tooltip : '{pop}: x={x}{xunit}; y={y}%; clients={clients}; n={n}';
```

Comment:
This is actually the number of pings that have a non-null value for this particular metric and cohort.

Reply:
Hmm. Is there another term that comes to mind that could be used here?

Reply:
This is actually going to be related to #274 after #273 lands in the API, I think. In the last EV meeting it was mentioned that you cannot simply multiple the percentage by the total pings.